### DISCUSSION TITLE
[cargo-smart-release] custom tags

### DISCUSSION THREAD

**Discussion by apepkuss** [GitoxideLabs/gitoxide#568] [Q&A]
I'm learning `cargo-smart-release`, and it's a great tool for me. I have a question: can I specify a custom tag for a release? Thanks a lot!

**Comment by Byron** [comment]
Thanks a lot, I am happy to hear that!

Custom tags are not supported as versions in tags are significant. Note that it always uses annotated tags and these could have meta-data, so in theory it's possible to add support for custom tags, even though I don't think it would be easy.
